schema/sql/schema.sql

```diff
@@ -97,6 +97,14 @@ create table "ssh_public_key" (
   "uid" integer references passwd (uid) on delete cascade
 );
 
+create domain pgp_sha256_fingerprint bytea check (length(value) = 64);
+
+create table "gpg_public_key" (
+  "uid" integer references passwd (uid) on delete cascade,
+  "fingerprint" pgp_sha256_fingerprint not null,
```

Comment:
Can this be automatically generated by an insert to the PostgREST endpoint in a fashion similar to the SSH keys?

Reply:
I'm unsure. We'll need to test this out.

Reply:
Dropping field for now

Reply:
This should be able to be implemented as soon as we figure out the bug with pgcrypto.pgp_key_id. However, this would require a version of postgres to be released with the bug fixed.